src/oscar/apps/order/models.py

```diff
@@ -31,53 +16,4 @@ class ShippingAddress(AbstractShippingAddress):
     class BillingAddress(AbstractBillingAddress):
         pass
 
-    __all__.append('BillingAddress')
-
-
-if not is_model_registered('order', 'Line'):
-    class Line(AbstractLine):
-        pass
-
-    __all__.append('Line')
-
-
-if not is_model_registered('order', 'LinePrice'):
-    class LinePrice(AbstractLinePrice):
-        pass
-
-    __all__.append('LinePrice')
-
-
-if not is_model_registered('order', 'ShippingEvent'):
-    class ShippingEvent(AbstractShippingEvent):
-        pass
-
-    __all__.append('ShippingEvent')
-
-
-if not is_model_registered('order', 'ShippingEventType'):
-    class ShippingEventType(AbstractShippingEventType):
-        pass
-
-    __all__.append('ShippingEventType')
-
-
-if not is_model_registered('order', 'PaymentEvent'):
-    class PaymentEvent(AbstractPaymentEvent):
-        pass
-
-    __all__.append('PaymentEvent')
-
-
-if not is_model_registered('order', 'PaymentEventType'):
-    class PaymentEventType(AbstractPaymentEventType):
-        pass
-
-    __all__.append('PaymentEventType')
-
-
-if not is_model_registered('order', 'OrderDiscount'):
-    class OrderDiscount(AbstractOrderDiscount):
-        pass
-
-    __all__.append('OrderDiscount')
+    __all__.append('BillingAddress')
```

Comment:
Why cant we move/remove all contents of this module?

Reply:
We could, but I wanted to move the simple 1000 LOCs before making internal changes.
The BillingAddress model comes from another app and this superior class is used multiple times and I thought I'd work on this in a new branch.

Reply:
D'accord.